examples/simple-average/run.py

```diff
@@ -15,31 +14,27 @@
   tfe.set_config(config)
   tfe.set_protocol(tfe.protocol.Pond())
 
-
+@tfe.local_computation
 def provide_input() -> tf.Tensor:
   # pick random tensor to be averaged
   return tf.random_normal(shape=(10,))
 
+@tfe.local_computation('result-receiver')
```

Comment:
maybe keep the same syntax as for `provide_input`, and give the player name when the function is called instead

Reply:
I preferred this way because it demonstrates both ways of using it. Maybe it would be best to add a comment that emphasizes that, or we can do that in different examples (just not sure we should expect people to pick up details from all examples)